VehicleIdentificationMsg identifiers
---
VehicleStatusMsg vehicle_status
GeographicalPositionMsg position
